### PR TITLE
Gesture Control Robot

### PR DESCRIPTION
**Pull request by ManikShrivastav**
## Describe your project in 1-3 sentences

Gesture Robot is a robotic car controlled by the movement of circuit.It is a car that can be controlled according to the user’s movement of the transmitter circuit.

## Hack Club Slack handle

manikshrivastav163

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: ManikShrivastav.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Name: “ManikShrivastav”
+Project: “Gesture Robot”
+---
+
+# Project Name
```
Comment (exu3) [inline ManikShrivastav.md:6]: Could you replace this title with the name of your project?

File: ManikShrivastav.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+|  2.  | NRF24L01 RF Module                    |  https://amzn.to/3BXDfFC              | 7.86   |      
+|  3.  | L298N Motor Driver                    |  https://amzn.to/3WqvbFK              | 4.12   |
+|  4.  | ADXL335 Module                        |  https://amzn.to/3HZsmaa              | 8.23   |
+|  5.  | TT Gear Motor                         |  https://amzn.to/3I2uMF8              | 3.15   |
+|  6.  | Rubber Wheels                         |  https://amzn.to/3hO8MmD              | 2.30   |
+|  7.  | 2.5mm Male Female Strip PCB Connector |  https://amzn.to/3Vlfj6c              | 2.06   |
+|  8.  | 18650 li-ion Battery                  |  https://amzn.to/3C1zwqF              | 10.04  |
+|  9.  | 18650 battery Holder(2 Slot)          |  https://amzn.to/3WIi3eR              | 1.80   |
+|  10. | Screw terminal (2 pin)                |  https://amzn.to/3vdaHnW              | 2.06   |
+|  11. | Beard Board                           |  https://amzn.to/3vgWEhi              | 2.30   |
```
Comment (exu3) [inline ManikShrivastav.md:38]:
```suggestion
|  11. | Breadboard                           |  https://amzn.to/3vgWEhi              | 2.30   |
```

File: ManikShrivastav.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+## Summary
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to it. After that, the motors will be connected with motor shield driver. After that, the Arduino Nano is
```
Comment (exu3) [inline ManikShrivastav.md:18]: > four motors and rubber wheels attached to it

Attached to what? I'm not sure if I missed something—what do you plan to make the frame out of?

File: ManikShrivastav.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+connected to the RF module and it acts as a transmitter with the power supply of li-ion battery. The
+other Arduino Nano is connected with RF Module and ADLX335 Module and it acts as a receiver with 
+power supply from 9V Battery. The code would be uploaded to both the Arduinos to make them 
+transmit and receive signals and then the circuit works.
+
+## Budget
+
+| SN   | Product                               | Supplier/Link                         | Cost   |
+| ---- | ------------------------------------- |-------------------------------------- | ------ |
+|  1.  | Arduino Nano                          |  https://amzn.to/3GiAeTd              | 20.46  |
```
Comment (exu3) [inline ManikShrivastav.md:28]:
```suggestion
|  1.  | 2x Arduino Nano                       |  https://amzn.to/3GiAeTd              | 20.46  |
```

It sounds like you'll be needing 2 of these?

File: ManikShrivastav.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+|  3.  | L298N Motor Driver                    |  https://amzn.to/3WqvbFK              | 4.12   |
+|  4.  | ADXL335 Module                        |  https://amzn.to/3HZsmaa              | 8.23   |
+|  5.  | TT Gear Motor                         |  https://amzn.to/3I2uMF8              | 3.15   |
+|  6.  | Rubber Wheels                         |  https://amzn.to/3hO8MmD              | 2.30   |
+|  7.  | 2.5mm Male Female Strip PCB Connector |  https://amzn.to/3Vlfj6c              | 2.06   |
+|  8.  | 18650 li-ion Battery                  |  https://amzn.to/3C1zwqF              | 10.04  |
+|  9.  | 18650 battery Holder(2 Slot)          |  https://amzn.to/3WIi3eR              | 1.80   |
+|  10. | Screw terminal (2 pin)                |  https://amzn.to/3vdaHnW              | 2.06   |
+|  11. | Beard Board                           |  https://amzn.to/3vgWEhi              | 2.30   |
+|  12. | Acrylic sheet (25cm*15cm)             |  https://amzn.to/3YKAmlr              | 3.27   |
```
Comment (exu3) [inline ManikShrivastav.md:39]: What will the acrylic be used for?

**Review by exu3 — CHANGES_REQUESTED** [review]
Thanks for submitting this @ManikShrivastav!

Could you update the summary/plan to describe the project more thoroughly? Specifically, I'm looking for where each item you included in your budget will be used.

File: ManikShrivastav.md (hunk 3 above)
Comment (ManikShrivastav) [reply ManikShrivastav.md:18]: I am planning to make a frame our of Plywood which I already own and cutting them into a required shape is a simple job for me!

File: ManikShrivastav.md (hunk 4 above)
Comment (ManikShrivastav) [reply ManikShrivastav.md:28]: Yes, one for the robot and one for controlling the robot remotely.

File: ManikShrivastav.md (hunk 5 above)
Comment (ManikShrivastav) [reply ManikShrivastav.md:39]: I am planning to make a transparent outer cover to protect the internal Circuit so it for that purpose.

File: ManikShrivastav.md (hunk 2 above)
Comment (ManikShrivastav) [reply ManikShrivastav.md]: Done!

File: ManikShrivastav.md (hunk 1 above)
Comment (ManikShrivastav) [reply ManikShrivastav.md:6]: Done!

**Comment by ManikShrivastav** [comment]
I have updated the plan, hope that describes the project and the used components!

File: ManikShrivastav.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Gesture Robot is an electronic car controlled by the movement of circuit. It can be mostly used in real 
+life robot fights and car fights. It is a car that can be controlled according to the user’s movement of the 
+hands. The transmitter part will be attached to the hand of the user and its receiver will act as the 
+motherboard of the car. As the transmitter moves, it will send the signals to the receiver in the car and it 
+will turn and run the car according to the instructions.
+
+## Plan
+
+First of all, the chassis of car is going to be made. It will contain four motors and rubber wheels attached 
+to the frame of the robot made out of PlyWood. After that, the motors will be connected with motor shield driver.
```
Comment (exu3) [inline ManikShrivastav.md:20]: Do you already have plywood you plan to use for this project? I don't see it in your budget.

Comment (ManikShrivastav) [reply ManikShrivastav.md:20]: Yes, I already own a plywood for making the project.

Comment (exu3) [reply ManikShrivastav.md:20]: Great.

Comment (exu3) [reply ManikShrivastav.md:20]: Once you do so, please request me for another review. It's almost good to go!

Comment (ManikShrivastav) [reply ManikShrivastav.md]: Done adding plywood to the budget.

**Comment by ManikShrivastav** [comment]
I have made all the change in the file requested.

**Review by exu3 — APPROVED** [review]
Nice work! The @hackclub/bank-ops team will reach out for next steps on receiving your grant through Hack Club Bank. 

Looking forward to seeing what you make!

**Review by MelSmith104 — COMMENTED** [review]
hey @ManikShrivastav! this sounds so cool! it sounds like it will be so much fun to build and to actually play around with! im dm'ing you on slack right now so that we can chat about next steps!

**Comment by MelSmith104** [comment]
hi @ManikShrivastav can you please let me know what your slack ID is? the one that you including in your proposal isn't popping up for me.

**Comment by ManikShrivastav** [comment]
Hello Ma'am, My slack I'd is @manikshrivastav163. Hope you see you on slack.

**Comment by ManikShrivastav** [comment]
Namaste @MelSmith104 Ma'am. My Slack I'd is @manikshrivastav163. Hope to see you soon in slack.

**Comment by MelSmith104** [comment]
sounds good! dm'ing you now :) thank you!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**